t/admin/global-rules.t

```diff
@@ -218,8 +218,8 @@ GET /t
 --- request
 GET /t
 --- error_code: 400
---- response_body
-{"error_msg":"invalid configuration: invalid \"additionalProperties\" in docuement at pointer \"#\/host\""}
+--- response_body_like eval
+qr/{"error_msg":"invalid configuration:/
```

Comment:
we lost the detail error info?

Reply:
different json schema will throw different error message, we do not want to check the detail info.